### PR TITLE
Scroll to a machine when navigating from status

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #3153.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- connect to a model with multiple machines
- open the status view
- click on a machine
- if your window is small enough that the machine/container list scrolls then the list should be scrolled enough to show that machine

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8038/

File: jujugui/static/gui/src/app/components/machine-view/machine-view.js [hunk 1]
```diff
@@ -230,25 +231,53 @@ class MachineView extends React.Component {
...
+    }
+  }
+
+  /**
     Handle selecting a machine.
 
-    @method selectMachine
     @param {String} id The machine id to select. If the id is null, no machine
       is selected.
+    @param {Boolean} dontScroll Whether to scroll to the selected machine.
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/machine-view/machine-view.js]: scrollToMachine

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8040/

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**